Makefile

```diff
@@ -214,7 +214,7 @@ all-local: $(ALL) ## Build all artifacts in the build tree
 $(ISO_ROOT)/bootstrap.sh: scripts/bootstrap.sh.in $(ISO_ROOT)/product.txt
 	mkdir -p $(shell dirname $@)
 	rm -f $@
-	sed s/@VERSION@/$(shell source $(ISO_ROOT)/product.txt && echo $$SHORT_VERSION)/g < $< > $@ || (rm -f $@; false)
+	sed s/§VERSION/$(shell source $(ISO_ROOT)/product.txt && echo $$SHORT_VERSION)/g < $< > $@ || (rm -f $@; false)
```

Comment:
Huh?

Reply:
See commit message https://github.com/scality/metalk8s/pull/795/commits/87564c9d38803d1fc713af79152ae410d225ca5d

I could have used `@{placeholder}` but then `shellcheck` complains.

Reply:
I, for one, don't have it on my keyboard, which makes editing a bit cumbersome, at least...

Reply:
Then it's annoying indeed. I'll try to find something in the ASCII charset that won't conflict easily and won't trigger `shellcheck`.

Reply:
It seems that I can use a string as delimiter (for some reason I thought that I was limited to a single char) so let's use `@@VERSION` instead of `@VERSION@`.

(thanks @gdemonet for opening my eyes)